tests/src/Test/Tasty/Clash.hs

```diff
@@ -129,6 +130,8 @@ data TestOptions =
     , verilate :: Verilate
     -- ^ Whether to run compatible tests through verilator as well as / in
     -- place of the simulator that would ordinarily be used.
+    , vivado :: Bool
+    -- ^ Whether to simulate via Vivado (such as when one depends on Xilinx IP)
```

Comment:
That makes sense.

Reply:
It suddenly randomly occurred to me it might be much less work if in addition to `hdlSim` (still as a `Bool`) we additionally define `hdlSimWith` with a list of tools. This saves changing the 121 lines in `Main.hs` that say `hdlSim=False`.

Reply:
I like `hdlSim` as a list because we can take `null hdlSim` to deduce the boolean we had before (and this enforces that everything is the same).

Reply:
Sure, I think it's fine.  `data HdlSim = DefHdlSim | HdlSim [Sim] | NoHdlSim` might be even more intuitive.

Reply:
Ok, I think I'll keep with a list of `Sim` for now, those cases can be handled with the current setup.

Reply:
I think `HdlSim []` and `NoHdlSim` would have the same meaning, which doesn't sit well to me. Doing the `null` check seems the way to go to me!

Reply:
Haha yes I suppose it would! I was just going for easy readability.